src/index.js

```diff
@@ -0,0 +1,5 @@
+import types from './types'
+import infer from './infer'
+import Schema from './schema'
+
+module.exports = { infer, types, Schema }
```

Comment:
It seems a simple change ( `types` -> `TypeGuesser` ) so I'd do it now. if not, sure, just open a ticket so we remember to do it next :)